examples/platforms/cc1352/radio.c

```diff
@@ -1575,14 +1575,18 @@ otError otPlatRadioAddSrcMatchExtEntry(otInstance *aInstance, const otExtAddress
 {
     OT_UNUSED_VARIABLE(aInstance);
 
-    otError error = OT_ERROR_NONE;
-    uint8_t idx   = rfCoreFindExtSrcMatchIdx((uint64_t *)aExtAddress);
+    otError  error = OT_ERROR_NONE;
+    uint8_t  idx;
+    uint64_t extAddress;
+
+    memcpy(&extAddress, aExtAddress, sizeof(extAddress));
```

Comment:
This could be done with a cast and assignment to a uint64_t. I think GCC is smart enough to optimize out the function call, so that is probably fine.

Reply:
Yes, cast-and-assignment seems to work here as well.